x/staking/handler_test.go

```diff
@@ -78,7 +78,7 @@ func TestValidatorByPowerIndex(t *testing.T) {
 
 	// slash and jail the first validator
 	consAddr0 := sdk.ConsAddress(PKs[0].Address())
-	app.StakingKeeper.Slash(ctx, consAddr0, 0, initPower, sdk.NewDecWithPrec(5, 1))
+	app.StakingKeeper.Slash(ctx, consAddr0, 0, initPower, sdk.NewDecWithPrec(5, 1), 0)
```

Comment:
Is it okay to have `0` here as the infraction type? I've seen it also in `x/staking/keeper/slash_test.go`.

Reply:
Actually proto imposes to have a `0` numeric value to define it as the default value. (fyi [proto guide](https://developers.google.com/protocol-buffers/docs/proto3#enum))

In these context, knowing that the `InfractionType` argument is disabled in the SDK `Slash` function:
```
func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec, _ types.InfractionType)
```  
 
the `0` default value, named `InfractionEmpty`, should indicates that it won't be used.